v0/destinations/branch/transform.js

```diff
@@ -1,4 +1,5 @@
 const get = require("get-value");
+const { isEmpty } = require("lodash");
```

Comment:
Since what you need is `isNotEmpty`, import it from utils

Reply:
Okay